projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts

```diff
@@ -5,6 +5,8 @@ import { By } from '@angular/platform-browser';
 import { MockRpxTranslatePipe } from '../../../../../test/mock-rpx-translate.pipe';
 import { QueryListItem } from '../../models';
 import { QueryDetailsComponent } from './query-details.component';
+import { ActivatedRoute } from '@angular/router';
+import { Constants } from '../../../../../commons/constants';
```

Comment:
Minor comment - Mentioned this on earlier PR but external imports should be above internal ones